docs/software/container-development-guide.md

```diff
@@ -162,6 +162,15 @@ Images that have passed acceptance testing should be tagged as `stable`:
 
 1. Get the sha256 repo digest of the image that the user has tested.
    All you need is the part that starts with `sha256:...` (aka the `<DIGEST>`).
+   
+   A Kubernetes user can get the digest by from the "Image" and "Image ID" lines obtained by running:
```

Comment:
>by from

Maybe just "from" ?

> "Image" and "Image ID"

Does the user need to combine these lines or something?  Maybe it's obvious from the output when they run it?

Reply:
Yeah I'll use just "Image ID".